sdk/src/types/block/mod.rs

```diff
@@ -45,19 +43,17 @@ pub mod signature;
 pub mod slot;
 /// A module that provides types and syntactic validations of unlocks.
 pub mod unlock;
-/// A module that provides types and syntactic validations of validation blocks.
-pub mod validation;
 
 pub(crate) use r#macro::create_bitflags;
 #[cfg(feature = "serde")]
 pub(crate) use r#macro::{impl_id, string_serde_impl};
 
 #[cfg(feature = "serde")]
-pub use self::core::dto::BlockDto;
+pub use self::core::dto::{BlockDto, BlockWrapperDto};
```

Comment:
Not really sure why this is all under core now instead of just under block

Reply:
I'd like to have them isolated together, easier to follow. But I'm open to other names if core is the issue.